tools/ops/policylambda.py

```diff
@@ -56,79 +53,94 @@ def render(p):
     if key_arn:
         properties['KmsKeyArn']
 
-    if p.execution_mode == 'periodic':
-        # Event render
-        revents = {}
-        revents = {
-            'PolicySchedule': {
-                'Type': 'Schedule',
-                'Properties': {
-                    'Schedule': p.data.get('mode', {}).get('schedule')}}
-        }
-        properties['Events'] = revents
-
-    elif p.execution_mode == 'cloudtrail':
-        events = [e for e in policy_lambda.get_events(None)
-                if isinstance(e, mu.CloudWatchEventSource)]
-        if not events:
-            return
-
-        revents = {}
-        for idx, e in enumerate(events):
-            revents[
-                'PolicyTrigger%s' % string.ascii_uppercase[idx]] = {
-                    'Type': 'CloudWatchEvent',
-                    'Properties': {
-                        'Pattern': json.loads(e.render_event_pattern())}
-            }
-        properties['Events'] = revents
+    return properties
 
-    elif p.execution_mode == 'config-rule':
-        properties.pop('Delay', None)
 
-    return {
-        'Type': 'AWS::Serverless::Function',
-        'Properties': properties}
+def render_periodic(p, policy_lambda, sam):
```

Comment:
`render` seems like a strange word to use when you are just returning a dict.

Reply:
render is a transformation of an input format to an output format. there's no implication the rendered output is for human consumption or is not a data format. serialization would also work, but in many cases the iaac language outputs would potentially be intended for human edits.